### PR TITLE
Introducing Team Alpha Ruby

### PR DESCRIPTION
**Pull request by teamalpharuby**
Initial blog post for Rails Girls Summer of Code - Introducing Team Alpha Ruby

### CODE REVIEW

File: blog/_posts/2015-08-16-Introducing-Team-Alpha-Ruby.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+title: Introducing Team Alpha Ruby (α ❤ ✌)
+layout: post
+created_at: Sun Aug 16 2015
+permalink: blog/2015-07-30-introducing-alpha-ruby
```
Comment (alicetragedy) [inline blog/_posts/2015-08-16-Introducing-Team-Alpha-Ruby.md]: please change the permalink date to the date of your post (`2015-08-16`)

File: blog/_posts/2015-08-16-Introducing-Team-Alpha-Ruby.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Hello everybody! We are [Team Alpha Ruby](https://teams.railsgirlssummerofcode.org/teams/83), Ariane and Ramya - no team name pun intended ;-).
+
+During Rails Girls Summer of Code 2015, we will be working on the [Official Ruby Documentation Redesign](http://docs.ruby-lang.org/)
+
+Our team has wonderful support on this journey from our mentor Zachory Scott and our coaches Barbara Barbosa and Stella Miranda.
+Furthermore Benedikt Deicke provides additional input and help as our team’s supervisor.
+
+<img src="/img/blog/2015/introducing-team-alpha-ruby.jpg" alt="Team Alpha Ruby" width="600">  
+<font color="grey"><small><i>The amazing members of Team Alpha Ruby</i></small></font>
```
Comment (alicetragedy) [inline blog/_posts/2015-08-16-Introducing-Team-Alpha-Ruby.md]: We just need an image credit here (like who took the picture/who the image belongs to). See other teams' posts if you're having trouble, but usually the credit was given to the team or to a single team member. :)

**Comment by alicetragedy** [comment]
Great post @teamalpharuby! I've just made a couple of comments of minor things that need changing/adding. Please push those changes as soon as you can, after that I will be able to merge and publish your post! :sparkles:

**Comment by teamalpharuby** [comment]
Hi @alicetragedy 

Thank you for the comments :-)

We have made the appropriate changes.

**Comment by alicetragedy** [comment]
perfect, it's ready to go! I will merge it and publish tomorrow and tweet about it. Thank you for submitting the PR early enough so we could review it! :sparkling_heart: and have a great weekend @teamalpharuby!

**Comment by teamalpharuby** [comment]
Hi @alicetragedy Could you please also add the latest commit, that includes a spelling fix.

Thank you :-)

**Comment by Aumrr** [comment]
Hi @alicetragedy Thank You

**Comment by alicetragedy** [comment]
@teamalpharuby done! Congratulations on your first successful pull request on our  `summer-of-code` repo! :tada: :confetti_ball:

**Comment by Aumrr** [comment]
@alicetragedy Thank you , Onbehalf of TeamAlphaRuby

**Pull request merged**